Client/Frontend/Shields/SiteReportedView.swift

```diff
@@ -0,0 +1,57 @@
+// Copyright 2020 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import UIKit
+import Shared
+import BraveShared
+
+class SiteReportedView: UIStackView, Themeable {
```

Comment:
What do you think about auto dismissing this view after few seconds?

Reply:
Just tested desktop and it actually dismisses automatically after 5s, so I'll add auto-dismissal too, but 5s seems too long for mobile, going to shorten it